static/src/stylesheets/module/_adslot.scss

```diff
@@ -8,6 +8,10 @@
     }
 }
 
+.aside-slot-container {
+    min-height: 300px;
```

Comment:
Would min-height be 274px instead?

This is the height of a MPU (250px) plus the advertisement label (24px)